### ISSUE TITLE
User Story:  Complete the errata process for RHEL 8 discovery cli (0.9.1) and discovery-server (0.9.1) rpms

### ISSUE THREAD

**Issue by kholdaway** [redhat-discovery/discovery#192]
CLI Advisory: https://errata.devel.redhat.com/advisory/45700
Server Advisory: https://errata.devel.redhat.com/advisory/44409

**Comment by myersCody** [comment]
We were noticed a failure for TPS testing for the discovery-server rpm because of brotli on 09/18/19. Therefore, we opened this issue to research wether or not brotli is an actual runtime requirement for the discovery-server ("quipucords"): https://github.com/redhat-discovery/discovery/issues/205

The spike revealed that we could remove it from the runtime requirements. Therefore, we opened this pr: https://github.com/redhat-discovery/discovery/pull/206

It was officially built again today: 
```
Task Info: https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=23581850
Build Info: https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=971229
```
And the the advisory has been updated for the server, and once the RPM Diff is fixed it will be flipped back to the QE state.
